tools/php_api_ref/.phpdoc/template/trait.html.twig

```diff
@@ -0,0 +1,89 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}{{ node.name }} | {{ parent() }}{% endblock %}
+{% block meta %}
+    <meta name="description" content="{{ node.summary|escape }}" />
+    {{ parent() }}
+{% endblock %}
+
+{% block content %}
+    {% include 'components/breadcrumbs.html.twig' %}
+    {% include 'components/trait-title.html.twig' %}
+    {% include('components/element-header.html.twig') %}
```

Comment:
```suggestion
    {% include 'components/element-header.html.twig' %}
```

Reply:
https://twig.symfony.com/doc/3.x/tags/include.html recommends to use https://twig.symfony.com/doc/3.x/functions/include.html but maybe for another time. Let's just unify this block for now.